vendor/modules.txt

```diff
@@ -457,6 +457,10 @@ github.com/huaweicloud/golangsdk/openstack/vbs/v2/policies
 github.com/huaweicloud/golangsdk/openstack/vbs/v2/tags
 github.com/huaweicloud/golangsdk/openstack/vpcep/v1/endpoints
 github.com/huaweicloud/golangsdk/openstack/vpcep/v1/services
+github.com/huaweicloud/golangsdk/openstack/waf/v1/certificates
+github.com/huaweicloud/golangsdk/openstack/waf/v1/domains
```

Comment:
please update the file by `go mode vendor` as github.com/huaweicloud/golangsdk/openstack/waf/v1/domains is redundant.

Reply:
has changed to waf_hw